### DISCUSSION TITLE
Strange Lines

### DISCUSSION THREAD

**Discussion by ghost** [openstreetmap/iD#10245] [Q&A] [answered]
Hi there ! Over the past few days, the lines on the editor have become strange (almost invisible). Is this normal?
![image](https://github.com/openstreetmap/iD/assets/116185937/bfd5ad09-24f2-405b-838e-9e6ef9c44bf9)

**Comment by riQQ** [comment, marked as answer]
You pressed <kbd>W</kbd>.

**Reply by ghost** [reply]
Thanks ^^
